index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Ramiro Aquistapace</title>
+    <link rel="stylesheet" href="style.css" />
+    <style>
+      @import url("https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap");
+    </style>
+</head>
+<body>
+    <div class="wrapper">
+      <header class="header">
+        <h3 class="mylogo">My Logo</h3>
+        <img alt="" src="Union.png" class="icon" />
+      </header>
+      <div class="container">
+        <h1>I’m Ramiro Glad to see you!</h1>
+        <p>
+          I’m a software developer and a incredible Micronaut! I can help you
+          build a product , feature or website Look through some of my work and
+          experience! If you like what you see and have a project you need
+          coded, don’t hestiate to contact me.
+        </p>
+        <h2 class="connect">LET’S CONNECT</h2>
+        <img alt="" src="social-media-icons.png" class="social-media" />
```

Comment:
- [ ] The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag) (Currently you just added them all together as an image, please change them to single items aligned with ul and li tags)